backend/grant-service/src/main/kotlin/grant/services/Applications.kt

```diff
@@ -502,6 +503,44 @@ class ApplicationService(
         )
     }
 
+    suspend fun transferApplication(
+        ctx: DBContext,
+        actor: Actor,
+        currentProject: String?,
+        applicationId: Long,
+        transferToProjectId: String
+    ) {
+        val application =  viewApplicationById(ctx, actor, applicationId)
+        if (application.first.grantRecipient !is GrantRecipient.ExistingProject) {
+            if (currentProject.isNullOrEmpty()) {
+                throw RPCException.fromStatusCode(
+                    HttpStatusCode.BadRequest,
+                    "Not in a project. No applications in Personal workspace"
```

Comment:
Is there any reason we couldn't just use the source project? I don't understand why the user's active project _needs_ to match the source of funds. We can just use the source available in the application.